### PR TITLE
add cli in base images.

### PR DESCRIPTION
**Pull request by droslean**

### CODE REVIEW

File: ci-operator/config/openshift/origin/openshift-origin-release-4.3.yaml [hunk 1]
```diff
@@ -12,6 +12,10 @@ base_images:
     name: "4.3"
     namespace: ocp
     tag: machine-os-content
+  cli:
+    name: "4.2"
```
Comment (petr-muller) [inline ci-operator/config/openshift/origin/openshift-origin-release-4.3.yaml]: 4.3

Comment (droslean) [reply ci-operator/config/openshift/origin/openshift-origin-release-4.3.yaml]: addressed, thanks

**Comment by droslean** [comment]
/close

Closing, #4465

**Pull request closed without merging**